sys/include/sema.h

```diff
@@ -58,7 +68,6 @@ typedef struct {
  * @param[in] value Initial value for the semaphore.
  *
  * @return  0 on success.
- * @return  -EINVAL, if semaphore is invalid.
```

Comment:
Here and below: please document the assertions you make in the implementation using the `@pre` doxygen command.